### ISSUE TITLE
"Execution failed because the return code was nonzero"

### ISSUE THREAD

**Issue by thuy-nguyenthanh** [cms-dev/cms#1157]
I'm using CMS v 1.4
When submitting a file with a code that is simply printed the float numeric value, it encounters an error:  "Execution failed because the return code was nonzero "

Code:

```
Using System;
Namespace exp
{
    Class Program
    {
        static int Main ()
        {
            Console. Write (0.55 f);
            return 0;
        }
    }
}
```

Please support,
Thanks!

**Comment by andreyv** [comment]
Most likely you are getting an exception. #1134 improves evaluation messages.

Try running the same program manually on the server, using the same compiler (I guess `mcs`) and the same compilation parameters (visible when submitting). Also you can enable `keep_sandbox` in the config, get sandbox path from CMS logs, and then examine the log file in the sandbox.

**Comment by andreyv** [comment]
I believe this is fixed in 10d8712a85185fafc71d92e80e6c5dc06701cc82, as I could reproduce the issue only without this commit.
